lms/lib/comment_client/user.py

```diff
@@ -180,6 +185,15 @@ def retire(self, retired_username):
             metric_tags=self._metric_tags
         )
 
+    def delete(self):
+        hash = hex(int(time.time()))[2:]
```

Comment:
@dyudyunov please rename the var, `hash` is a reserved name

Reply:
thanks, I'll update it